### DISCUSSION TITLE
React (Tanstack) Location

### DISCUSSION THREAD

**Discussion by PointSingularity** [salvoravida/redux-first-history#86] [Q&A] [answered]
First things first, thanks for the great library!

I was thinking of maybe using [React Location](https://react-location.tanstack.com/) instead of React Router.

I was wondering if someone had successfully integrated Redux First History and Redux Toolkit with React Location?

Any insight would be greatly appreciated!

**Comment by salvoravida** [comment, marked as answer]
Since react-location use history from react-router it already works with redux-first-history

Here is the demo

https://codesandbox.io/s/redux-first-history-demo-rr6-forked-wvst19

**Reply by PointSingularity** [reply]
This is great, thank you!
Thanks for the example too, I really appreciate it 👍

**Reply by salvoravida** [reply]
just added react-location to the homepage. Thanks for the info.

**Reply by PointSingularity** [reply]
No, thank you!
